app/src/main/java/org/p2p/wallet/auth/model/Web3AuthSignInResponse.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.wallet.auth.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Web3AuthSignInResponse(
+    @SerializedName("ethPublic") val ethereumPublicKey: String,
```

Comment:
Let's set `@SerializedName` annotation above the parameter, not on the same line.
If you format this class it would make the formatting